examples/codebert_example.rs

```diff
@@ -0,0 +1,120 @@
+use std::path::PathBuf;
+fn feature_extraction() -> Result<()> {
+    //    Set-up model
+    let config = CodeBertForFeatureExtractionConfig::new(
+        ModelType::CodeBert,
+        RemoteResource::from_pretrained(CodeBertModelResources::CODEBERT),
+        None,
+        RemoteResource::from_pretrained(CodeBertConfigResources::CODEBERT),
+        RemoteResource::from_pretrained(CodeBertVocabResources::CODEBERT),
+        RemoteResource::from_pretrained(CodeBertMergesResources::CODEBERT),
+        true,
+        None,
+        None,
+    );
+
+    let feature_extraction_model = CodeBertForFeatureExtractionModel::new(config)?;
+    //    Define input
+    let input = ["this is an example sentence", "each sentence is converted"];
+
+    //    Run model
+    let output = feature_extraction_model.predict(&input);
+    for hidden_states in output {
+        println!("{:?}", hidden_states);
+    }
+
+    Ok(())
+}
+
+fn masked_language() -> Result<()> {
+    let config = MaskedLanguageConfig::new(
+        ModelType::CodeBert,
+        RemoteResource::from_pretrained(CodeBertModelResources::CODEBERT_MLM),
+        // None,
+        LocalResource {
+            local_path: PathBuf::from("/home/vincent/.cache/model/codebert-base-mlm/rust_model.ot"),
+        },
+        RemoteResource::from_pretrained(CodeBertConfigResources::CODEBERT_MLM),
+        RemoteResource::from_pretrained(CodeBertVocabResources::CODEBERT_MLM),
+        Some(RemoteResource::from_pretrained(
+            CodeBertMergesResources::CODEBERT_MLM,
+        )),
+        true,
+        None,
+        false,
+    );
+    //    Set-up model
+    let model = MaskedLanguageModel::new(config)?;
+    //    Define input
+    let input = [
+        "Looks like one <mask> is missing!",
+        "The goal of life is <mask>.",
+    ];
+
+    //    Run model
+    let output = model.predict(&input, vec![5, 6]);
+    for word in output {
+        println!("{:?}", word);
+    }
+
+    Ok(())
+}
+
+fn sequence_classification() -> Result<()> {
+    let config = SequenceClassificationConfig::new(
+        ModelType::CodeBert,
+        RemoteResource::from_pretrained(CodeBertModelResources::CODEBERTA_LANG),
+        None,
+        RemoteResource::from_pretrained(CodeBertConfigResources::CODEBERTA_LANG),
+        RemoteResource::from_pretrained(CodeBertVocabResources::CODEBERTA_LANG),
+        RemoteResource::from_pretrained(CodeBertMergesResources::CODEBERTA_LANG),
+        true,
+        None,
+        false,
+    );
+    //    Set-up model
+    let sequence_classification_model = SequenceClassificationModel::new(config)?;
+    //    Define input
+    let input = [
+        "const foo = 'bar'",
+        "Toutcome := rand.Intn(6) + 1",
+        "def f(x): return x**2",
+    ];
+
+    //    Run model
+    let output = sequence_classification_model.predict(&input);
+    for label in output {
+        println!("{:?}", label);
+    }
+
+    Ok(())
+}
+
+fn main() {
```

Comment:
Can you please ensure we only run relevant example (providing valid outputs)? General tests for model functionality can be moved to integration tests